device/config.py

```diff
@@ -89,6 +89,7 @@ class Config:
     # --------------
     uiport: int = get_toml('webui_settings', 'uiport', 5432)
     uitheme: str = get_toml('webui_settings', 'uitheme', 'dark')
+    twilighttimes: str = get_toml('webui_settings', 'twilighttimes', False)
```

Comment:
Your types don't agree here. I think if you hit the default, you may end up with type errors. I think you want to change `str` to `bool`

Reply:
Whoopie! Good catch, thanks.